dockstore-webservice/src/main/java/io/dockstore/webservice/languages/CWLHandler.java

```diff
@@ -736,7 +736,10 @@ public VersionTypeValidation validateWorkflowSet(Set<SourceFile> sourcefiles, St
                 validationMessage = "Primary descriptor is empty.";
             } else if (!content.contains("class: Workflow")) {
                 isValid = false;
-                validationMessage = "Requires class: Workflow.";
+                validationMessage = "A CWL workflow requires class: Workflow.";
+                if (content.contains("class: CommandLineTool") || content.contains("class: ExpressionTool")) {
+                    validationMessage += " This file contains class: CommandLineTool or ExpressionTool. Did you mean to register a tool?";
```

Comment:
Suggest mentioning the exact class(es) rather than the the possible choices.

Reply:
Fixed so the message shows the exact class.

CommandLineTool:
![CWL CommandLineTool Error Message](https://user-images.githubusercontent.com/25287123/73942225-b2db7600-48bc-11ea-9216-87fb0e322262.png)

ExpressionTool:
![CWL ExpressionTool Error Message](https://user-images.githubusercontent.com/25287123/73942279-c4248280-48bc-11ea-843c-d8b27394cafc.png)